examples/gno.land/p/demo/grc/grc721/grc721.gno

```diff
@@ -0,0 +1,261 @@
+package grc721
+
+import (
+	"gno.land/p/demo/avl"
+	"gno.land/p/demo/ufmt"
+	"std"
+)
+
+type Token struct {
+	name              string
+	symbol            string
+	owners            *avl.Tree // tokenID > std.Address
+	balances          *avl.Tree // std.Address > # of owned tokens
+	tokenApprovals    *avl.Tree // tokenID > std.Address
+	operatorApprovals *avl.Tree // "OwnerAddress:OperatorAddress" -> bool
+	tokenURIs         *avl.Tree // tokenID > URI
```

Comment:
since you've at least 3 elements sharing the same key, I suggest having a single avl.Tree but returning an object.